bot/resources/tags/sql-fstring.md

```diff
@@ -0,0 +1,16 @@
+**SQL & f-strings**
+Don't use f-strings (`f""`) or other forms of "string interpolation" (`%`, `+`, `.format`) to inject data into a SQL query. It is an endless source of bugs and syntax errors. Also, in user-facing applications, it represents a major security risk via SQL injection.
```

Comment:
Isn't "presents" rather than "represents"?

My avatar represents me. On the other hand a broken door presents a security risk.